docs/guide/connecting.rst

```diff
@@ -7,30 +7,97 @@
 Connecting to Verilated Models
 ******************************
 
+Structure of the Verilated Model
+================================
+
+Verilator outputs a :file:`{prefix}.h` header file which defines a class
+named :code:`{prefix}` which represents the generated model the user is
+supposed to instantiate.  This Model Class defines the interface of the
+Verilated model.
+
+Verilator will additionally create a :file:`{prefix}.cpp` file, together
+with additional .h and .cpp files for internals.  See the :file:`examples`
+directory in the kit for examples.  See :ref:`Files Read/Written` for
+information on all the files Verilator might output.
+
+The output of Verilator will contain a :file:`{prefix}.mk` file that may be
+used with Make to build a :file:`{prefix}__ALL.a` library with all required
+objects in it.
+
+The generated Model Class file manages all internal state required by the
+model, and exposes the following interface that allows interaction with the
+model:
+
+
+* Top level IO ports are exposed as references to the appropriate internal
+  equivalents.
+
+* Public top level module instances are exposed as pointers to allow access
+  to :code:`/* verilator public */` items.
+
+* The root of the design hierarchy (as in SystemVerilog :code:`$root`) is
+  exposed via the :code:`rootp` member pointer to allow access to model
+  internals, including :code:`/* verilator public_flat */` items.
+
+
+.. _Porting Post 4.204:
```

Comment:
```suggestion
.. _Porting from Pre-Verilator 4.204:
```

Maybe we should have a later section near DEPRECATIONS that describes porting and cross reference this to there?

Reply:
I think we should use "pre 4.206", assuming that will be the next version?

Reply:
Yup.

Reply:
BTW, this is just an anchor, the only thing it influences is URLs that link to it.

Reply:
Ah, yes fine, I misread that.